### ISSUE TITLE
Character name parsing errors

### ISSUE THREAD

**Issue by mwspace** [Tagirijus/fountain#9]
Character name parsing fails with these naming scenarios:

```
BILL (O.S.)
C-3PO
GROUPIE #1
SARAH'S FRIEND
BILL / THE MONSTER
MICHAEL (as DOROTHY)
```

All of these work as expected in Highland2 (a Fountain editor by John August). All punctuation appears to be acceptable in character names, and lowercase characters are acceptable if enclosed in parentheses.

**Comment by Tagirijus** [comment]
Thanks for reporting. I will have a look at some point. At the moment, unfortunately, my life is quite ... busy. Thanks for your understanding and patience!

**Comment by Tagirijus** [comment]
Solved with PR #10 ! Thanks @satamame ! (=
